.github/workflows/stale.yml

```diff
@@ -0,0 +1,21 @@
+name: 'Close stale issues'
+on:
+  schedule:
+    - cron: '0 */1 * * *'
```

Comment:
Runs every hour, this should atm mark 300 issues and close those after 14 days so it'd take an entire day to do so due to rate limiting